src/js/components/Calendar/__tests__/__snapshots__/Calendar-test.js.snap

```diff
@@ -2487,14 +2487,14 @@ exports[`Calendar children 1`] = `
             className="c5"
             size="medium"
           >
-            September 2020
+            October 2020
```

Comment:
This change will be fixed once merging in with the master branch.

Reply:
Ok, I'll remove it.